app/wama.web/Views/AdminConsole/UserTool/CreateNewUserAccount.cshtml

```diff
@@ -60,6 +60,16 @@
             <label asp-for="Gender"></label>
             <select asp-for="Gender" asp-items="@Html.GetEnumSelectList<Gender>()" class="form-control"></select>
         </div>
+        @{if ((ViewBag.AccountType) == UserAccountType.Patron)
```

Comment:
I see what you're trying to do there. Typically, this type of logic shouldn't be exposed to the end user.

> **_Always remember that any critical input that is being received from a user, has to be received with a grain of salt_**